components/users/usersService.js

```diff
@@ -41,7 +41,9 @@ class usersService {
       if (regExpId.test(args[criteria])) {
         return { [criteria]: parseInt(args[criteria]) };
       } else {
-        return { [criteria]: args[criteria] };
+        return {
+          [criteria]: { $regex: new RegExp(`^${args[criteria]}`, 'i') },
```

Comment:
```js
new RegExp(`.*${args[criteria].*}`, 'i') }
```
 for substrings

Reply:
Test RegExp with criteria numbers for reduce all to one case

Reply:
Okay! Thanks @hyfi06 for you advices!

Reply:
I changed for this:

new RegExp(`.*${args[criteria]}.*`, 'i') }

Regards!